DSpace/DSpace
# DS-3240: comm/coll admins misses some functions and a propper navbar.

https://jira.duraspace.org/browse/DS-3240

To test this: 
1. Create a community/collection and a community/collection admin. 
2. Create and publish/archive an item.
3. Login as community/collection admin, open the item and edit it. The admin navbar should appear and contain fields like "General Settings", that are protected from being used by community/collection admins.
4. Make the item private or withdraw it.
5. Choose from the admin navbar (still as community/collection admin) the button "content/withdrawn items" or "content/private items". You won't see the Item you just made private.

With this PR you should get a admin navbar containing only functions community/collection admins should be able to use. Also you should be able to see private/withdrawn item as community/collection admin.
